docs/advanced.rst

```diff
@@ -127,6 +127,29 @@ referencing the ``changed_by`` field:
 
 Admin integration requires that you use a ``_history_user.setter`` attribute with your custom ``_history_user`` property (see :ref:`admin_integration`).
 
+Custom ``history_id``
+---------------------
+
+If you want to use something other then an ``AutoField`` for the history
```

Comment:
"By default, the historical table of a model will use an `AutoField` for the table's `history_id` (the history table's primary key). However, you can specify a different type of field for `history_id` but passing a different field to `history_id_field` parameter." Something like that sounds a little more clear to me